bin/storm.py

```diff
@@ -1401,7 +1401,7 @@ def drpc_client(args):
 
     exec_storm_class(
         "org.apache.storm.command.BasicDrpcClient",
-        args=remove_common_options(sys.argv[2:]), storm_config_opts=args.storm_config_opts,
+        main_class_args=remove_common_options(sys.argv[2:]), storm_config_opts=args.storm_config_opts,
```

Comment:
why do we need to change this

Reply:
This is just to make the name more descriptive